lib/helpers.php

```diff
@@ -149,7 +153,7 @@ function micropub_post($endpoint, $params, $access_token, $file_path = NULL, $js
     $mimetype = finfo_file($finfo, $file_path);
     $multipart = new p3k\Multipart();
     $multipart->addArray($params);
-    $multipart->addFile($file_prop, $file_path, $mimetype);
+    $multipart->addFile($file_prop, $filename, $mimetype, $file_content);
```

Comment:
wow, so that parameter was added a year ago. I bet I was looking at the local copy of the library in the `vendor` folder which wasn't updated for that release! Looks like I'm going to need to publish a new release of `php-multipart-encoder` and this PR will need to require greater than that version.